src/msg/async/dpdk/DPDK.cc

```diff
@@ -774,23 +789,9 @@ bool DPDKQueuePair::rx_gc(bool force)
   if (_num_rx_free_segs >= rx_gc_thresh || force) {
     ldout(cct, 10) << __func__ << " free segs " << _num_rx_free_segs
                    << " thresh " << rx_gc_thresh
-                   << " free pkts " << _rx_free_pkts.size()
+                   << " free pkts " << _rx_free_bufs.size()
```

Comment:
nit, the ldout message may be "free bufs"

Reply:
ok,thanks